### ISSUE TITLE
Packages: Move `elements` to its own package

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#8776]
## Task Description

The idea for this dedicated `elements` package would be to provide various APIs to register & retrieve element types. But _not_ the actual individual element types supported by the editor.

See https://docs.google.com/document/d/1ImRe9wYKWGCAZiq3jro5KLpSJK_NMKXHC6FWqs3nwl0/edit for context.
See also `@wordpress/blocks` for context.

**Comment by spacedmonkey** [comment]
> But not the actual individual element types supported by the editor.

Would be have a elements-library package for this?

**Comment by swissspidy** [comment]
> > But not the actual individual element types supported by the editor.
> 
> Would be have a elements-library package for this?

See #8777. It's marked as blocked by this ticket.

**Comment by swissspidy** [comment]
Duplicate of #8125
